ha/core/event_analyzer/filter/filter.py

```diff
@@ -86,3 +104,52 @@ def filter_event(self, msg: str) -> bool:
         except Exception as e:
             raise EventAnalyzerError(f"Failed to filter event. Message: {msg}, Error: {e}")
 
+
+class IEMFilter(Filter):
+    """ Filter IEM consumed by watcher """
+
+    def __init__(self):
+        """
+        Init method
+        """
+        super().__init__()
+        # Loads IEM filter rules in the configuration
```

Comment:
Move this to `Filter` class as rule file same for both IEM and Alert

Reply:
Agree

Reply:
Yes, done.